backend/config/accounts.go

```diff
@@ -30,8 +30,13 @@ type Account struct {
 	//
 	// This is used to unify multiple Bitcoin script types (p2wsh, p2wsh-p2sh) in one account. The
 	// keystore must be able to sign transactions with mixed inputs.
-	SupportsUnifiedAccounts bool                   `json:"supportsUnifiedaccounts"`
-	Configurations          signing.Configurations `json:"configurations"`
+	SupportsUnifiedAccounts bool `json:"supportsUnifiedaccounts"`
```

Comment:
I wonder why add this to the persistent config. By the time a keystore is connected/opened, which root fingerprint is how its accounts are identified by, you should be able to tell whether unified accounts are supported by calling `keystore.SupportsUnifiedAccounts()`.

I see a challenge in `Backend.initPersistedAccounts()` because it doesn't actually get a hold on the active keystore and probably needs some refactoring. Not sure what I'm missing but it seems to me unnecessary to persist something that is easly known at runtime?

Reply:
This is with lookahead  to watch-only. Then, the connecting a keystore will become optional (resp. only required to receive/send). So we need to persist all info from the keystore that we need.

Reply:
But from what I understand by looking around, "unified accounts" support is needed only during signing, i.e. a connected keystore. Why would you care about it in watch-only mode, only for UI?

Reply:
> (resp. only required to receive/send)

resp?

Reply:
Well yes, there is no other way really. If you see the accounts as split with the keystore connected, you expect to see them the same way in watch-only.

Besides, if it is loaded and shown as a unified account, the balance is unified and the inputs are mixed when sending. I don't see a way to undo that only when sending without extreme confusion.

Reply:
:+1:

Reply:
> resp?

respectively